module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2048,6 +2056,11 @@ protected function getItemStatusesForBiblio($id, $patron = null, array $options
             return [];
         }
 
+        // Get the total number of items returned from the API call and set it to
+        // a class variable. It is then used in VuFind\RecordTab\HoldingsILS for
```

Comment:
I'm not sure what "set it to a class variable" refers to in this comment. You appear to be using a local array rather than a class property.

Reply:
Thanks @demiankatz I've included that fix in my follow up.